src/components/TabsSlider/TabsSlider.Overview.stories.mdx

```diff
@@ -143,3 +143,38 @@ A common use case for the TabsSlider as a presentational toggle switch, along wi
     }}
   </Story>
 </Canvas>
+
+## With Anchor elements
+
+Since each `Tab.Item` is render via a `Box` component you can pass the `as` attribute to render it as a different element than a button.
```

Comment:
```suggestion
Since each `TabSlider.Item` is render via a `Box` component you can pass the `as` attribute to render it as a different element than a button.
```